etl/steps/data/garden/AI/2023-06-07/monmoth_poll.py

```diff
@@ -0,0 +1,47 @@
+"""Load a meadow dataset and create a garden dataset."""
+
+from typing import cast
+
+import pandas as pd
+from owid.catalog import Dataset, Table
+from structlog import get_logger
+
+from etl.helpers import PathFinder, create_dataset
+
+log = get_logger()
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+def run(dest_dir: str) -> None:
+    log.info("monmoth_poll.start")
+
+    #
+    # Load inputs.
+    #
+    # Load meadow dataset.
+    ds_meadow = cast(Dataset, paths.load_dependency("monmoth_poll"))
+
+    # Read table from meadow dataset.
+    tb = ds_meadow["monmoth_poll"]
+    df = pd.DataFrame(tb)
+    df["answer"] = df["answer"].str.replace(r"\(VOL\) ", "", regex=True)
+    columns_to_rename = df.columns.difference(["year", "answer"])
+    new_column_names = [f"Q{i+1}" for i in range(len(columns_to_rename))]
```

Comment:
If you have between 10 and 100 questions, it'd be better to name columns like `f"Q{i+1:02d}" for i ...`.